libs/cardano-ledger-core/CHANGELOG.md

```diff
@@ -2,20 +2,29 @@
 
 ## 1.19.0.0
 
+* Added:
+  - `ppMinFeeConstantCompactL`,
+  - `ppuMinFeeConstantCompactL`,
+* Renamed:
+  - `hkdMinFeeAL` -> `hkdMinFeeFactorL`
+  - `ppMinFeeAL` -> `ppMinFeeFactorL`
+  - `ppuMinFeeAL` -> `ppuMinFeeFactorL`
+  - `hkdMinFeeBL` -> `hkdMinFeeConstantL`
+  - `ppMinFeeBL` -> `ppMinFeeConstantL`
+  - `ppuMinFeeBL` -> `ppuMinFeeConstantL`
+* Changed type to `CoinPerByte`:
+  - `hkdMinFeeAL`
+  - `ppMinFeeAL`
+  - `ppuMinFeeAL`
+* Added `CoinPerByte` (moved from `cardano-ledger-babbage`)
 * Changed name and type to `CompactForm Coin`:
-  - `hkdMinFeeAL` -> `hkdMinFeeACompactL`
-  - `hkdMinFeeBL` -> `hkdMinFeeBCompactL`
```

Comment:
As we didn't have a release since these changes, I opted to remove these entries (and some more both here and in other `CHANGELOG`s) since they were made obsolete by this PR.